src/i18n/DA/grid-resources.ts

```diff
@@ -0,0 +1,133 @@
+    igx_grid_filter_lessThanOrEqualTo: 'Mindre end eller lig med',
+    igx_grid_filter_true: 'True',
+    igx_grid_filter_false: 'False',
+    igx_grid_filter_all: 'Alle',
+    igx_grid_filter_condition_placeholder: 'Vælg filter',
+    igx_grid_summary_count: 'Optælling',
+    igx_grid_summary_min: 'Min',
+    igx_grid_summary_max: 'Maks',
+    igx_grid_summary_sum: 'Sum',
+    igx_grid_summary_average: 'Gns',
+    igx_grid_summary_earliest: 'Tidligst',
+    igx_grid_summary_latest: 'Senest',
+    igx_grid_excel_filter_moving_left: 'Flyt til venstre',
+    igx_grid_excel_filter_moving_left_short: 'venstre',
+    igx_grid_excel_filter_moving_right: 'Flyt til højre',
+    igx_grid_excel_filter_moving_right_short: 'højre',
+    igx_grid_excel_filter_moving_header: 'flyt',
+    igx_grid_excel_filter_sorting_asc: 'stigende',
+    igx_grid_excel_filter_sorting_asc_short: 'stig',
+    igx_grid_excel_filter_sorting_desc: 'aftagende',
+    igx_grid_excel_filter_sorting_desc_short: 'aft',
+    igx_grid_excel_filter_sorting_header: 'sortér',
+    igx_grid_excel_filter_clear: 'Ryd kolonnefiltre',
+    igx_grid_excel_custom_dialog_add: 'tilføj filter',
+    igx_grid_excel_custom_dialog_clear: 'Ryd filter',
+    igx_grid_excel_custom_dialog_header: 'Brugerdefineret autofilter på kolonne ',
+    igx_grid_excel_cancel: 'annuller',
+    igx_grid_excel_apply: 'anvend',
+    igx_grid_excel_search_placeholder: 'Søg',
+    igx_grid_excel_select_all: 'Vælg alle',
+    igx_grid_excel_select_all_search_results: 'Vælg alle søgeresultater',
+    igx_grid_excel_add_to_filter: 'Tilføj aktuel markering til filter',
+    igx_grid_excel_blanks: '(Blanke)',
+    igx_grid_excel_hide: 'Skjul kolonne',
+    igx_grid_excel_show: 'Vis kolonne',
+    igx_grid_excel_pin: 'Fastgør kolonne',
+    igx_grid_excel_unpin: 'Frigør kolonne',
+    igx_grid_excel_select: 'Vælg kolonne',
+    igx_grid_excel_text_filter: 'Tekstfilter',
+    igx_grid_excel_number_filter: 'Talfilter',
+    igx_grid_excel_date_filter: 'Datofilter',
+    igx_grid_excel_boolean_filter: 'Boolsk filter',
+    igx_grid_excel_currency_filter: 'Valutafilter',
+    igx_grid_excel_custom_filter: 'Brugerdefineret filter',
+    igx_grid_advanced_filter_title: 'Avanceret filtrering',
+    igx_grid_advanced_filter_and_group: '"Og" gruppe',
+    igx_grid_advanced_filter_or_group: '"Eller" gruppe',
+    igx_grid_advanced_filter_end_group: 'Afslut gruppe',
+    igx_grid_advanced_filter_create_and_group: 'Opret "Og" gruppe',
+    igx_grid_advanced_filter_create_or_group: 'Opret "Eller" gruppe',
+    igx_grid_advanced_filter_and_label: 'og',
+    igx_grid_advanced_filter_or_label: 'eller',
+    igx_grid_advanced_filter_add_condition: 'Tilstand',
+    igx_grid_advanced_filter_ungroup: 'Fjern gruppe',
+    igx_grid_advanced_filter_delete: 'Slet',
+    igx_grid_advanced_filter_delete_filters: 'Slet filtre',
+    igx_grid_advanced_filter_initial_text: 'Start med at oprette en tilstandsgruppe, der er knyttet til 'Og' eller 'Eller'',
```

Comment:
Can fix this build error for the quotes,  the same way as it was in the IT, using double quotes
Ex: igx_grid_advanced_filter_initial_text: 'Inizia con la creazione di un gruppo di condizioni collegate con "And" o "Or"',